package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "test",
+  "version": "1.0.0",
+  "description": "Let's do test",
+  "main": "test.js",
+  "scripts": {
+    "test": "mocha tests/testSumOfOther.js tests/testMake.js tests/testRecursion.js",
+    "eslint": "eslint src/sumOfOther.js src/make.js src/recursion.js || true"
+  },
+  "env": [ "mocha" ],
+  "author": "",
+  "devDependencies": {
+    "eslint": "^5.8.0",
+    "eslint-config-airbnb": "^17.1.0",
```

Comment:
Eslint-config-airbnb-base and eslint-config-airbnb are not equal.
Eslint-config-airbnb contains all of our ESLint rules, including ECMAScript 6+ and React. <...> If you don't need React, see eslint-config-airbnb-base. https://www.npmjs.com/package/eslint-config-airbnb